spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 # Create the share API context
 # so we can pass stuff between
 # the different tests
-RSpec.shared_context "shared environment", :a => :b do
+RSpec.shared_context 'shared environment', :a => :b do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.